### ISSUE TITLE
Yolo_nas_s size after training 

### ISSUE THREAD

**Issue by tjasmin111** [Deci-AI/super-gradients#1661]
### 💡 Your Question

I have a script that trains a nas small model on a large dataset of 40,000 images (I borrowed the scripts from [here](https://www.google.com/amp/s/blog.roboflow.com/yolo-nas-how-to-train-on-custom-dataset/amp/?espv=1)). 

First, it is too slow to train. It's taking like 24 hrs and I see it reached epoch 46 on an A100. Is it normal and the way it should be?

Second, the size of produced models is 250MB while the original nas_s model was 76MB. Shouldn't it stay the same after training?

**Comment by BloodAxe** [comment]
The size of saved checkpoint file should not confuse you since it also contains optimizer state (it is used if you decide to resume experiment)

As for training speed - you may want to monitor GPU usage and try to maximize it by increasing batch size and use more workers in DataLoaders. 

I cannot provide support on roboflow example since it's not officially supported tutorial.
My suggestion is not to use colab/notebooks for any serious training and use recipes we provide instead and build your recipe on top of that

**Comment by tjasmin111** [comment]
I see, Thanks. And btw, how can I select the NAS_S_INT8 model? Looks like it's faster than s.

`MODEL_ARCH = 'yolo_nas_s'`

**Comment by BloodAxe** [comment]
A quantized weights are not available since there is no easy way into storing them in state dict. So if you want to have quantized model - you need to load  regular model first and then do trainer.ptq or trainer.qat as show in https://colab.research.google.com/github/Deci-AI/super-gradients/blob/master/notebooks/yolo_nas_custom_dataset_fine_tuning_with_qat.ipynb and then export quantized model to onnx.
